ajax/libs/numbers.js/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "author":"Steve Kaliski <sjkaliski@gmail.com> (https://github.com/sjkaliski)",
```

Comment:
Could you please put `author` right below the `filename`, we prefer to put `name` and `filename` in the head of the package.json. Thanks!